### PR TITLE
Blesses the codebase with Blahaj

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request

Adds the Shark and Shork Costume to the game, craftable via carp scales.
In addition, adds the ability to use a shark costume to craft a shark plushy.

ge1uucdAYn8xAAAAABJRU5ErkJggg
J2YyKAAI2pNXkXCAyMAAI0sAajuhAYE4FeBciufHkQc53Vij1rxorYmNyadxkKAQRo3lII0FBclnqOiQAChACNyZ95l4ER6FWAbFjkcSrxraha5Q7MF6guBHongADNkZcQvt5bkwIhMDACCBACNDCXpbpjItCrAI0JHO8CAQikE0CA0hliAQIQ6EgAAeoIjmwQgEA6gf8Bgzt4GlTrNLUAAAAASUVORK5CYII

It is five carp scales and five leather to craft the shark costume, take the costume and craft it with wirecutters to make a shork costume. Take a shark costume, 10 cotton, and five cloth to craft a shark plush.

## Why It's Good For The Game

Good and funny costumes that give an additional use to pre-existing crafting materials = good
Giving players another way to craft the shark plushie = good

Todo:
- [x] Make sure shit wont break
- [x] Add the crafting recipe for the costume
- [x] Add the crafting recipe for the shark plushie
- [x] Profit??

## Changelog

:cl: Drag
add: Adds the Shark and Shork costume. Blahaj lovers rejoice!
/:cl:

### CODE REVIEW

**Comment by another-expler** [comment]
Fqmboy update

**Review by san7890 — CHANGES_REQUESTED** [review]
I like the idea but you screwed up some code so I'll mention it now

**Comment by Admiral-Gigaton** [comment]
Wuh... which is uh... the shork...

**Comment by Draggeru** [comment]
> Wuh... which is uh... the shork...

Shork is the very fucked up MS paint looking costume

File: code/modules/clothing/suits/costume.dm [hunk 1]
```diff
@@ -362,13 +362,55 @@
...
+	flags_inv = HIDEHAIR|HIDEEARS
+
 /obj/item/clothing/suit/hooded/bloated_human //OH MY GOD WHAT HAVE YOU DONE!?!?!?
 	name = "bloated human suit"
 	desc = "A horribly bloated suit made from human skins."
 	icon_state = "lingspacesuit"
 	icon = 'icons/obj/clothing/suits/costume.dmi'
 	worn_icon = 'icons/mob/clothing/suits/costume.dmi'
-	inhand_icon_state = "labcoat"
+	inhand_icon_state = "sharkcursed"
```
[the remark this answers is not in the document]
Comment (Draggeru) [reply code/modules/clothing/suits/costume.dm:413]: Officially hate it here

**Comment by Draggeru** [comment]
After vibe checking my imagination with my skill level instead of it being a random chance it is now just a craft to get the shork costume.

File: code/modules/clothing/suits/costume.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/clothing/suits/costume.dm:413]:
```suggestion
	inhand_icon_state = "labcoat"
```

**Comment by Wallemations** [comment]
Show shark plushie icon in pr please

**Comment by Draggeru** [comment]
> Show shark plushie icon in pr please

The plushie is the one that already exists in the code base

**Comment by Admiral-Gigaton** [comment]
> > Wuh... which is uh... the shork...
> 
> Shork is the very fucked up MS paint looking costume

Both... do...

**Comment by Jackraxxus** [comment]
Katy Perry left shark

**Comment by relativelysmalllebowski** [comment]
mistake

**Pull request merged**